docs/topics/selectors.rst

```diff
@@ -35,12 +35,12 @@ defines selectors to associate those styles with specific HTML elements.
     in speed and parsing accuracy to lxml.
 
 .. _BeautifulSoup: https://www.crummy.com/software/BeautifulSoup/
-.. _lxml: http://lxml.de/
+.. _lxml: https://lxml.de/
 .. _ElementTree: https://docs.python.org/2/library/xml.etree.elementtree.html
-.. _cssselect: https://pypi.python.org/pypi/cssselect/
-.. _XPath: https://www.w3.org/TR/xpath
+.. _cssselect: https://pypi.org/project/cssselect/
```

Comment:
`_cssselect` is defined multiple times. Here, earlier as `_cssselect: https://github.com/scrapy/cssselect/`, and apparently in parsel, which gives a weird (for me) build conflict: https://github.com/scrapy/parsel/issues/185

What might be the best way to resolve this?

Reply:
One way may be to use InterSphinx (https://github.com/scrapy/scrapy/blob/master/docs/conf.py#L282) for CSS select, and have those two references use something like :doc:`cssselect <cssselect:index>` instead.

Reply:
That seems sensible. I don't think anyone would mind the link change from the pypi & github pages to the docs, there.